### DISCUSSION TITLE
Welcome to raid-toolkit-sdk Discussions!

### DISCUSSION THREAD

**Discussion by dnchattan** [raid-toolkit/raid-toolkit-sdk#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WOROTH** [comment]
Hi everyone!
I'm developing a Scripted Bot for 'Raid', that can fully replace player on a daily routine.
RTK is a key component, that provides an actual ingame info for me.
It's a really powerful and useful tool, and i belive that we cam make it flawless together! ;)

**Comment by justuscook** [comment]
I making a discord bot and currently using the api to pull champion skill and stay information, and several more things as they are made available.  This is an awesome project and I can't wait to see it keep evolving.
